db/seeds.d/020-permissions_list.rb

```diff
@@ -34,10 +34,6 @@ def permissions
         ['ConfigReport', 'view_config_reports'],
         ['ConfigReport', 'destroy_config_reports'],
         ['ConfigReport', 'upload_config_reports'],
-        ['ConfigGroup', 'view_config_groups'],
-        ['ConfigGroup', 'create_config_groups'],
-        ['ConfigGroup', 'edit_config_groups'],
-        ['ConfigGroup', 'destroy_config_groups'],
```

Comment:
Can we separate PR to drop the permissions?
We will need to drop them from https://github.com/theforeman/foreman/blob/324bafdb625b06fe939530ef77534170be855447/app/registries/foreman/access_permissions.rb#L267 as well and we can't do it before both UI and API are in.